.editorconfig

```diff
@@ -0,0 +1,19 @@
+root = true
+
+[*]
+indent_style = space
+indent_size = 2
+end_of_line = lf
+charset = utf-8
+trim_trailing_whitespace = true
+insert_final_newline = true
+curly_bracket_next_line = false
+indent_brace_style = K&R
+
+[*.md]
+trim_trailing_whitespace = false
+
+[*.{java,gradle}]
+indent_size = 4
+continuation_indent_size = 2
+Number of imports required for '*' = 5
```

Comment:
J'ai l'impression que c'est une propriété, même si elle n'est pas similaire aux [autres](https://github.com/editorconfig/editorconfig/wiki/EditorConfig-Properties#ideas-for-domain-specific-properties)